### ISSUE TITLE
VTEX SECRETS

### ISSUE THREAD

**Issue by evander-corebiz** [vtex/faststore#2506]
Hello. I need to know if it is possible to use vtex secrets in the Ever Green version and how I can do this.

**Comment by carlosananias** [comment]
hi @evander-corebiz 
are you unable to work with keys with [vtex plugins install webops-secrets](https://v1.faststore.dev/how-to-guides/webops/security/setting-up-secrets)? I understand that it should still be valid for v2 of faststore.

**Comment by evander-corebiz** [comment]
Yeah, for v2 is enable, but we are using the Ever Green and it doesn't work,

**Comment by emersonlaurentino** [comment]
Evergreen now runs on webops, which supports secrets: https://developers.vtex.com/updates/release-notes/2025-06-24-faststore-webops-centralized-secrets-management
